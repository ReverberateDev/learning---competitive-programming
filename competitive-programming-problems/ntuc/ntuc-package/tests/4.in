1 70783315633526
-1
3028
