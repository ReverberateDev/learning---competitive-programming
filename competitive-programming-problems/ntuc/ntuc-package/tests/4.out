214331879738316728
